packages/diagrams/frontend/sirius-components-diagrams/package.json

```diff
@@ -43,6 +43,7 @@
     "@types/react": "18.3.3",
     "@types/react-dom": "18.3.0",
     "@xyflow/react": "12.2.1",
+    "react-draggable": "4.4.6",
```

Comment:
I really need to have dependencies updated properly so this change should include sirius-web-application, sirius-web-selection and sirius-web-papaya at least if I'm not mistaken.